.github/workflows/E2E-test.yml

```diff
@@ -14,7 +14,7 @@ on:
 jobs:
   test:
     # The type of runner that the job will run on
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
@bencourliss is it a good practice to use `ubuntu-latest`? Should we use a concrete ubuntu version?